backend-nodejs/expressApp/index.js

```diff
@@ -1,36 +1,39 @@
-const express = require('express');
-const cors = require('cors');
-const passport = require('passport');
-const bodyParser = require('body-parser');
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+
+const AWS = require("aws-sdk");
+const s3 = new AWS.S3({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_KEY,
+});
```

Comment:
@DonaldBrower does this work without manually inject credentials here?  I believe the aws sdk will use the environment variables if they are present.  This would allow other ways to use AWS credentials than forcing the environment variable method.

Reply:
is s3 being used in this file?